backends-common/postgres/src/main/java/org/apache/james/backends/postgres/utils/PostgresExecutor.java

```diff
@@ -39,13 +41,30 @@
 
 public class PostgresExecutor {
 
+    public static class Factory {
```

Comment:
That factory is great, would be useful in my current work :)

But I would have expected it to be used in the code though, and it seems it's not?

Reply:
It is used in mapper https://github.com/apache/james-project/pull/1808
But it takes a long time to merge